File: 1-TwoSum/1-TwoSum.cpp
// Last updated: 08/05/2026, 09:34:16
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int right=0;
        int n= nums.size();
        for(int left=0; left<n; left++){
            for(int right= left+1; right<n;right++){
                int diff=nums[right]+nums[left];
                if(diff==target){
                    return {left,right};
                }
            }
        }
        return {};
        
    }
};

File: 1-TwoSum/1-TwoSum.py
# Last updated: 08/05/2026, 09:34:17
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for idx, val in enumerate(nums):
                if target - val in nums[idx + 1:]:
                    return [idx, nums[idx + 1:].index(target - val) + (idx + 1)]
        # for i in range(len(nums)):
        #     if target-nums[i] in nums[i+1::]:
                
            
            
            # for j in range(i+1,len(nums)):
            #     if(nums[i]+nums[j]==target):
            #         return [i,j]
        